ui/app/AppLayouts/CommunitiesPortal/popups/CreateCommunityPopup.qml

```diff
@@ -145,14 +155,251 @@ StatusStackModal {
+                            filesList.selectedFiles.forEach(f => {
+                                files.splice(files.indexOf(f), 1)
+                            })
+                            filesList.selectedFiles = []
+                            return filesList.model = files
+                        }
+                        filesList.model = []
+                    }
+                }
+            }
+
+            FileDialog {
+                id: fileDialog
+
+                title: qsTr("Choose files to import")
+                selectMultiple: true
+                folder: shortcuts.pictures
+                nameFilters: [qsTr("JSON files (%1)").arg("*.json")]
+                onAccepted: {
+                    if (fileDialog.fileUrls.length > 0) {
+                        let files = []
+                        files.push(...fileDialog.fileUrls)
+                        filesList.model = files
+                    }
+                }
+            }
+        },
+
+        ColumnLayout {
+
+          id: categoriesAndChannelsView
+
+          readonly property bool canGoNext: root.store.discordChannelsModel.hasSelectedItems
+          property var prevAction: function () {
+            root.store.clearDiscordCategoriesAndChannels()
+            root.store.discordCategoriesAndChannelsExtractionFailed = false
+          }
+
+          Connections {
+              target: root.store
+              onDiscordCategoriesAndChannelsExtractionFailedChanged: {
+                  if (root.store.discordCategoriesAndChannelsExtractionFailed) {
+                    root.currentIndex--
+                    categoriesAndChannelsView.prevAction()
+                  }
+              }
+          }
+
+          Item {
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            visible: root.store.discordChannelsModel.count == 0
+            Loader {
+              anchors.centerIn: parent
+              active: parent.visible
+              sourceComponent: StatusLoadingIndicator {
+                width: 50
+                height: 50
+              }
+            }
+          }
+
+          ColumnLayout {
+              spacing: 12
+              visible: root.store.discordChannelsModel.count > 0
+
+              StatusBaseText {
+                  Layout.fillWidth: true
+                  text: qsTr("Please select the categories and channels to import into your new Status Community.")
+                  wrapMode: Text.WordWrap
+              }
+
+              StatusCheckBox {
+                  id: importAllHistoryCheckbox
+                  text: qsTr("Import all history")
+                  checked: true
+              }
+
+              StatusInput {
+                  label: qsTr("Import history from")
+                  input.text: new Date(root.store.discordOldestMessageTimestamp * 1000).toString()
+                  input.enabled: !importAllHistoryCheckbox.checked
+              }
```

Comment:
Would be cool to have a proper date picker here, but we don't even have designs for this flow yet. So I just made a simple input for now